### ISSUE TITLE
How to make screen readers announce new content? 

### ISSUE THREAD

**Issue by mikrogroove** [defunkt/jquery-pjax#548]
When using PJAX to load pages these are not announced by the NVDA screen reader, despite the PJAX container having both `aria-live="assertive"` and `aria-atomic="true"` attributes set. This makes the site unusable for NVDA users. What needs to be done to make screen readers read out the new content?

**Comment by mikrogroove** [comment]
Never mind, it seems I misunderstood how NVDA works; adding `aria-live="assertive"` and `aria-atomic="true"` does indeed solve the issue, but only when you use the keyboard to navigate, which I guess makes perfect sense.
